aspnetcore/web-api/action-return-types/samples/10/ControllerSSE/Controllers/HeartRateController.cs

```diff
@@ -0,0 +1,68 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Runtime.CompilerServices;
+using System.Net.ServerSentEvents;
+
+[ApiController]
+[Route("[controller]")]
```

Comment:
@mikekistler should I remove the controller sample? If not, where should I surface it?